### PR TITLE
Template and styles for search results

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Template and styles for search api

# Need Review By (Date)
- no rush

# Urgency
- low

# Steps to Test
1. see steps in https://github.com/SU-SWS/stanford_profile/pull/371

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: modules/stanford_profile_styles/lib/scss/base/views/search.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@charset "UTF-8";
+@import '../../config';
+
+.page-content {
+  h1 {
+    margin-top: 9.5rem;
```
Comment (sherakama) [inline modules/stanford_profile_styles/lib/scss/base/views/search.scss:6]: Our convention is to apply margin on the bottom of elements. Can this statement be moved to a different element with a margin-bottom instead?

File: stanford_profile_helper.module [hunk 2]
```diff
@@ -24,6 +24,17 @@ use Drupal\node\NodeInterface;
...
+/**
+ * Implements hook_theme().
+ */
+function stanford_profile_helper_theme($existing, $type, $theme, $path) {
+  $themes['block__stanford_basic_search'] = [
+    'template' => 'block--stanford-basic-search',
+    'original hook' => 'block',
+  ];
+  return $themes;
+}
```
Comment (sherakama) [inline stanford_profile_helper.module:36]: Why is this needed? Do the template suggestions not work here?

File: modules/stanford_profile_styles/lib/scss/base/views/search.scss (hunk 1 above)
Comment (pookmish) [reply modules/stanford_profile_styles/lib/scss/base/views/search.scss:6]: I just used the same approach from stanford_basic https://github.com/SU-SWS/stanford_basic/blob/8e445c84fda48bf969bb446c2843039f9fb79be2/src/scss/components/stanford-page/_page-title.scss#L8

File: stanford_profile_helper.module (hunk 2 above)
Comment (pookmish) [reply stanford_profile_helper.module:36]: not from within a module unfortunately.

**Comment by imonroe** [comment]
Tested in coordination with https://github.com/SU-SWS/stanford_profile/pull/371 but unfortunately the search form in the header doesn't seem to work.

File: modules/stanford_profile_styles/lib/scss/base/views/search.scss (hunk 1 above)
Comment (imonroe) [reply modules/stanford_profile_styles/lib/scss/base/views/search.scss:6]: Given the conversation in dev chat today, are we letting this go through?

Comment (pookmish) [reply modules/stanford_profile_styles/lib/scss/base/views/search.scss:6]: yeah i believe so.

**Review by imonroe — APPROVED** [review]
This works.  Tested with the changes here: https://github.com/SU-SWS/stanford_profile/pull/371
